src/app/content/components/Content.tsx

```diff
@@ -14,56 +14,10 @@ import Wrapper from './Wrapper';
 interface PropTypes {
   page: State['page'];
   book: State['book'];
-  loading: boolean;
```

Comment:
Why remove the loading state? I think it would be good to have so that Selenium tests have a way to wait until the new page has loaded. Also, people on slower connections might wonder if clicking did anything

Reply:
there is still a loading state in the redux state, this just wasn't doing anything here anymore so i removed it. when we have design for a loading state we can decide where to put it